bin/README_SHELL_COMPLETION

```diff
@@ -0,0 +1,32 @@
+# ossh is an ssh replacement.
+
+
+Ossh uses a dynamic inventory cache in order to lookup hostnames and translate them
```

Comment:
Ossh, should be ossh to follow the command name. Can be "The command ossh uses..."